lib/utils.js

```diff
@@ -0,0 +1,121 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
```

Comment:
Let us add a test for this module as well - I know it is being tested indirectly through other modules, but it is good to have a check for each module which isn't dependent on other modules.

Reply:
Don't you think this will make more redundant tests?
Because what will be tested is essentially already tested in the other modules?

Reply:
Hmm, we are actually not testing these functions right, we are testing some other functions which call these functions.

Reply:
Yes but the only way to test this is in that context. The functions need the regex that is used in the other modules.

Reply:
Hmm, why? We can pass mock contexts, filenames to check if the functions do the expected task, right?

Reply:
Yes we can.
Done.